pkg/client/clientcmd/api/types.go

```diff
@@ -83,6 +83,8 @@ type Context struct {
 	AuthInfo string `json:"user"`
 	// Namespace is the default namespace to use on unspecified requests
 	Namespace string `json:"namespace,omitempty"`
+	// NamespacePath is the path to a kubernetes ns file (~/.kubernetes_ns)
```

Comment:
+1 to nuking namespace-path. I don't think we have anything using it now, but removing it in a separate PR sgtm.